lib/kafo/kafo_configure.rb

```diff
@@ -193,13 +193,18 @@ def execute
         @progress_bar = config.app[:colors] ? ProgressBars::Colored.new : ProgressBars::BlackWhite.new
       end
 
-      unless skip_checks_i_know_better?
-        unless SystemChecker.check
-          puts "Your system does not meet configuration criteria"
+      if checks_only? || !skip_checks_i_know_better?
+        logger = Logger.new('checks')
+        if !SystemChecker.check
```

Comment:
Rubocop has a rule to prefer positive tests instead of negative tests. So write
```suggestion
        if SystemChecker.check
```

(and swap the branches)

Would that make more sense?